install.sh

```diff
@@ -177,6 +177,19 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  NVM_PROFILE=$(nvm_detect_profile)
+  if [ -z "$NVM_PROFILE" ] ; then
```

Comment:
What will this do on `sh`, `dash`, and `ksh`? Is there a way to figure out programmatically if `bash_completion` will run on the current shell?